Formula/postgresql@95.rb

```diff
@@ -0,0 +1,147 @@
+class PostgresqlAT95 < Formula
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v9.5.5/postgresql-9.5.5.tar.bz2"
```

Comment:
Ideally it's worth just doing both in a single PR, I think, but feel free to 👎 that.